gapis/api/vulkan/templates/vulkan_layer.tmpl

```diff
@@ -23,6 +23,7 @@
     {{Global "Vulkan.LayerNamespace" "<The namespace of your vulkan functions>"}}
     {{Global "Vulkan.OverrideFunctions" List Of Functions That Should Be Overriden}}
     {{Global "Vulkan.OtherUsedFunctions" List Of Functions That your layer will call down into}}
+    {{Global "Vulkan.ImplementedFunctions" List Of Functions That your layer will call down into}}
```

Comment:
This needs a slightly different description than the one above it. It should say that these are functions that should be intercepted even if the lower levels don't provide them, correct?

Reply:
Correct, done.